npm-shrinkwrap.json

```diff
@@ -340,7 +340,7 @@
     "fxa-auth-db-mysql": {
       "version": "1.105.0",
       "from": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#master",
-      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#0d874f9ae8b8415f838673e927d0be325022e596",
+      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#c22e19df6ba0eb6cd5f9f42121f8076ebfb6162f",
```

Comment:
don't forget to change this to https://github.com/mozilla/fxa-auth-db-mysql/commit/ee19e1bc432a27fac84b3a17fdeeba0a32c19107 ?